src/modules/Images360/Images360.js

```diff
@@ -142,13 +143,15 @@ export class Images360 extends EventDispatcher{
 
 	focus(image360){
 		if(this.manager.isFocussed()){
+			this.manager.showLinkedAnnotations(this.focusedImage,image360)
 			this.manager.unFocus(true);
 		}
 		else {
 			// When moving focus from one image to another, preserve the return position for the camera. Otherwise, set it from the current position.
 			previousView = {};
 		}
 		this.manager.setSelected360(this.parent);
+		this.manager.showLinkedAnnotations(this.focusedImage,image360)
 
 		this.focusedImage = image360;
 

```

Comment:
@saikrishanpadigapati333 I was getting some errors when trying to open the 360 view. Replacing lines 172-179 with the following seems to fix the issue:

		if(this.focusedImage.neighbors.includes(i) && pm) {
			const pointerSprite = new Sprite(pm);
			this.node.add(pointerSprite);
			this.pointers.push(pointerSprite);
			pointerSprite.sphere = otherMesh;
			
			const pointerScale = 0.1;
			const aspectRatio = ((image)=> { 
				if (image)
					image.width/image.height;
				})(pm.map.image);

Check and make sure this additional error handling works on your end. If so I will approve the PR

Reply:
@nathanturpin-powern could you add pin_white_2D.png in the local resources of potree in build file while running cpms-web.The error is pm being undefined.